### ISSUE TITLE
any method to set the listen backlog for ssl listener?

### ISSUE THREAD

**Issue by inter169** [emqtt/emqttd#1018]
#### Environment

- OS: centos7
- Erlang/OTP: 8.1
- EMQ: 2.1.1

#### Description

Hello,
 we need to export the ssl port (8883) to devices, so necessary to set a bigger backlog for this port,
 but I tried to set following configurations:
 mqtt.listener.ssl.backlog = 8192
 or
 mqtt.listener.ssl.tcp.backlog = 8192

it didn't work, the mqttd failed to start, how to set the backlog for SSL listener?

Thanks

**Comment by emqplus** [comment]
@inter169 Fixed in 2.2-beta.1 release. You can configure the the backlog for `tcp` or `ssl` listeners:

```
listener.tcp.external.backlog = 1024

listener.ssl.external.backlog = 1024

```

**Comment by inter169** [comment]
Thanks, it works on version v2.2 now
